content/tracing/advanced_usage/manual_tracing.md

```diff
@@ -33,6 +33,41 @@ public class MyClass {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+We have a Flask Python application that when called on `/doc` returns **42**.
+We instrumented our python code in order to generate traces from it:
+
+```python
+import time
+import blinker as _
+
+from flask import Flask, Response
+
+from ddtrace import tracer
+from ddtrace.contrib.flask import TraceMiddleware
+
+# Tracer configuration
+tracer.configure(hostname='datadog')
+app = Flask('API')
+traced_app = TraceMiddleware(app, tracer, service='doc_service')
```

Comment:
Too many details. This is about "Manual Tracing" and not how to use the Python APM. Also, with `ddtrace-run` we expect to avoid any manual change.